PostBoard.Api/Controllers/BirthdayController.cs

```diff
@@ -70,14 +66,15 @@ public IActionResult Update([FromRoute] int id, [FromBody] Birthday input)
             return BadRequest("Validation error.");
         }
 
-        var birthday = Birthdays.FirstOrDefault(b => b.Id == id);
+        var birthday =_dbcontext. Birthdays.FirstOrDefault(b => b.Id == id);
```

Comment:
Add space `' '` before `_dbContext` and remove one after. Should be:
`...birthday = _dbcontext.Birthdays...`